picking_good_slis.md

```diff
@@ -0,0 +1,86 @@
+# Picking Good SLIs
+
+## SRE's Philosophy on Monitoring
+
+A core goal of Site Reliability Engineering (SRE) is to ensure the availability of a service. "Availability" here means much more than simply "is my application running." Effectively monitoring a modern cloud application requires insight into the application's response time, error rate, traffic level, and more.
+
+Furthermore, the scope of monitoring should not be limited solely for use in deciding when to page an operations team. Metrics should be closely tied to business objectives, informing decisions on what features to prioritize and when to focus on adding new features versus improving reliability.
+
+To make this vision a reality, SRE teams rely on two key concepts:
+  * Service Level Indicators (SLIs) - A quantifiable metric that measures a specific aspect of a service's level of service.
```

Comment:
I am noticing that we define SLI/SLO several times in various docs.  I think once the dust settles we should go through and have either a single place where we define it or make sure all places where they're defined say the same exact thing. an #includes type of feature here would be nice ...